files/private-chef-ctl-commands/key-control.rb

```diff
@@ -0,0 +1,255 @@
+# Copyright (c) 2014 Chef Software, Inc.
+# All Rights Reserved
```

Comment:
Probably should be. Seems like we have a lot of copyright everywhere still in opscode-omnibus, but we should probably start putting the right things in new files.